crates/re_data_store/src/log_db.rs

```diff
@@ -38,7 +38,15 @@ impl Default for EntityDb {
                 InstanceKey::name(),
                 DataStoreConfig {
                     component_bucket_size_bytes: 1024 * 1024, // 1 MiB
-                    index_bucket_size_bytes: 1024,            // 1KiB
+                    // We do not garbage collect index buckets at the moment, and so the size of
+                    // individual index buckets is irrelevant, only their total number of rows
+                    // matter.
+                    // See <pr-link> for details.
```

Comment:
oh, i thought I pushed the commit with the link, but turns out i didnt press enter in the terminal :man_facepalming: